### DISCUSSION TITLE
OrcaSlicer now has an official Discord server!

### DISCUSSION THREAD

**Discussion by SoftFever** [OrcaSlicer/OrcaSlicer#1692] [Announcements]
Join the community
![discord-mark-blue](https://github.com/SoftFever/OrcaSlicer/assets/103989404/b97d5ffc-072d-4d0a-bbda-e67ef373876f)
https://discord.gg/P4VE9UY9gJ

**Comment by GiuseppeSiragusa** [comment]
elegoo OrangeStorm Giga ?????
